hw/ip/otp_ctrl/doc/_index.md

```diff
@@ -621,45 +688,48 @@ See further below for a detailed [Memory Map]({{< relref "#direct-access-memory-
 
 A typical readout sequence looks as follows:
```

Comment:
@mcy 
The updates below are mainly related to the `DIRECT_ACCESS_REGWEN` register, which is not SW writeable anymore now. I.e., this REGWEN is governed by HW now, and is set to 0 when the DAI is not idle in order to protect the associated CSRs during that period.